published/external/xdp/offload.h

```diff
@@ -143,8 +63,6 @@ typedef struct _XDP_FRAME_GRO_TIMESTAMP {
 
 C_ASSERT(sizeof(XDP_FRAME_GRO_TIMESTAMP) == 4);
 
-#pragma warning(pop)
```

Comment:
nit: in the future, put a comment on these pops to indicate what they are popping. It's unclear what you are deleting as a result.

Reply:
Agreed, they are confusing when unlabeled.